### ISSUE TITLE
Redesign bitcoin.design landing page

### ISSUE THREAD

**Issue by Bosch-0** [BitcoinDesign/Guide#925]
Gene recently brang up a discussion on [Slack](https://bitcoindesign.slack.com/archives/C015856BDME/p1663209728786449) around redesigning the bitcoin.design landing page. He made some visual suggestions in [this Figma file](https://www.figma.com/file/pgug8vW9yOa8dV619GtWnv/bitcoin.design-home-page-ideas?node-id=0%3A1). 

I've attempted to summarise the discussion below

- Too "we" centric, should be more "you" centric to make the community feel more inviting. Some suggestions he mentioned below
    -  “How you can play a role in bitcoin’s future…”
    - “Leave your mark on bitcoin - join …”
    - “Bitcoin needs you!” (meh)
- Talk about how this community is mostly design focused BUT it's also for anyone building or wanting to build a product. 
- Show off some achievements / profiles or community members - state how this could be "you" see first dot point
- Sell potential community members on the vision of being involved in this once-in-a-lifetime technology
- Make headline more active, currently too passive. 
- Add testimonials from community members. Adding faces could make it feel more human
- Make page more lively and fun
- Add an FAQ (link from landing page anyways, handy to have)
- Put more emphasis on the newsletter and guide which are two very key focal things which don't get the priority they deserve
- Show upcoming events

The overall end goal would be to make the landing page get people more involved by giving them a strong sense of responsibility, opportunity and possibilities.

Personally I agree with pretty much all of Gene's suggestions. The currently landing page really doesn't do the community justice. We could probably break some of these up into their own issues.

**Comment by tilltonystark** [comment]
Heya! just wanted to check if this issue is still open. If yes, then do I just need to redesign the landing page in my own figma file and share the link or is there any other or process or way to do it.

**Comment by GBKS** [comment]
Yes, this issue is still open. Keep in mind that there is no "official request" for a re-design. Gene simply suggested changes that can be made because he wanted to and felt that it improved the home page. Anyone can chime in to discuss, propose other changes, etc. There's no guarantee anything will be implemented, some issues are closed without action, sometimes they are implemented right away or a year later in a slightly different form, just depends on the community dynamics. Don't want to discourage you, just want to explain some of the dynamics.

A good way to start is by first sharing your design ideas to get some feedback. You can also hop straight into Figma if you'd like, it might just save you some time if you first check if people are generally on board with your ideas.

**Comment by peanutbutter003** [comment]
Hi @GBKS! I'd like to work on this, can I be assigned here?

**Comment by GBKS** [comment]
This came up again in [todays jam session](https://github.com/BitcoinDesign/Meta/issues/606). We only briefly looked at the home page, but the general sentiment is that the content can be improved. So I'm adding this to the roadmap for Q2.

**Comment by NishkarshG** [comment]
Hi @GBKS and @Bosch-0  is this issue still open.

Should I go to redesign this page?? in Figma first as per the requirements?
